pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -154,7 +204,24 @@ func TestRoundTripAndNewConnection(t *testing.T) {
 			t.Fatalf("%s: Error creating request: %s", k, err)
 		}
 
-		spdyTransport := NewRoundTripper(testCase.clientTLS)
+		spdyTransport := NewSpdyRoundTripper(testCase.clientTLS)
+
+		var proxierCalled bool
+		if testCase.withProxy {
+			proxyHandler := goproxy.NewProxyHttpServer()
+			proxy := testCase.serverFunc(proxyHandler)
```

Comment:
Is that because of separate certs, so that we can test both proxy and server with http/https independently?